src/components/topBar/topBar.js

```diff
@@ -51,8 +57,28 @@ class TopBar extends React.Component {
     this.dropdownRef = node;
   }
 
+  setSearchBarRef(node) {
+    this.searchBarRef = node;
+  }
+
+  onSearchClick() {
+    if (!this.state.isSearchDropdownEnable) {
+      document.addEventListener('click', this.onSearchClickOutside, false);
```

Comment:
Is it possible to change functions that are responsible for toggling dropdown to more generic, or move them to searchDropdownComponent?
In the future, there will be more than 1 dropdown available on topBar.
I would see it more like `isDropdownEnabled: { search: true, account: false, notifications: false }`, or even currentDropdownOpen: 'nameOfItemInMenu', because there will be always max 1 dropdown open and other will have to hide

Reply:
Maybe an implementation similar to the one used for the datepickers.
https://github.com/LiskHQ/lisk-hub/blob/development/src/components/transactionsV2/filters/dateFieldGroup.js